CONTRIBUTING.md

```diff
@@ -1,3 +1,540 @@
 # Contribution Guidelines
 
 We follow the [Contribution Guidelines of PySyft](https://github.com/OpenMined/PySyft/blob/dev/CONTRIBUTING.md), please refer to them to start developing!
```

Comment:
Q: Should we remove this?

Reply:
I am not sure, as you know I copied the structure and lot of parts from PySyft. I do not know if its nice to let it here for context. As you prefer, there is a link in the `Linking the PySyft src` section. So I think, it should be safe.